dotcom-rendering/src/web/components/PinnedPost.importable.tsx

```diff
@@ -92,14 +95,54 @@ export interface PinnedPostProps extends Props {
 
 export const PinnedPost = ({ pinnedPost, children }: PinnedPostProps) => {
 	const [expanded, setExpanded] = useState(false);
-	const isClient = typeof window !== 'undefined';
```

Comment:
IIRC this was preferred to adding a useEffect / state for the button to avoid potential re-renderings: https://github.com/guardian/dotcom-rendering/pull/4132#discussion_r817720427